Riot/Routers/NavigationRouterStore.swift

```diff
@@ -49,12 +49,12 @@ class NavigationRouterStore: NavigationRouterStoreProtocol {
         return navigationRouter
     }
     
-    // MARK: - Private
-    
-    private func findNavigationRouter(for navigationController: UINavigationController) -> NavigationRouterType? {
+    func findNavigationRouter(for navigationController: UINavigationController) -> NavigationRouterType? {
```

Comment:
I seem to think this was intentionally private and `navigationRouter(for navigationController: UINavigationController)` was meant to be used instead. @SBiOSoftWhare is that correct?

Reply:
Yes normally we should only use: `func navigationRouter(for navigationController: UINavigationController) -> NavigationRouterType`, it will create the `NavigationRouter` if it does not exist. 
`NavigationRouterStore` ensure there is only one `NavigationRouter` per `UINavigationController`. That's why we should not instantiate a `NavigationRouter` the outside of this store.

Reply:
I did it for the `RoomParticipantsInviteCoordinatorBridgePresenter.present()` method. I had a crash because `func navigationRouter(for navigationController: UINavigationController) -> NavigationRouterType` is creating a `NavigationRouter` with a new instance of `RiotNavigationController` and creating a `NavigationRouter` with the actual navigation controller was solving the crash.

Anyway, I cannot reproduce the crash...